docs/application/dotnet/guides/opentk/work-with-skiasharp.md

```diff
@@ -0,0 +1,183 @@
+```C#
+private void Create2DTextureFromMemory()
+{
+    GL.TexImage2D(
+                All.Texture2D,
+                0,
+                All.BgraExt,
+                bitmapWidth,
+                bitmapHeight,
+                0,
+                All.BgraExt,
+                All.UnsignedByte,
+                pBitMap
+            );
+
+    GL.TexParameter(All.Texture2D, All.TextureMinFilter, (float)All.LinearMipmapLinear);
+    GL.TexParameter(All.Texture2D, All.TextureMagFilter, (float)All.Linear);
+    GL.GenerateMipmap(All.Texture2D);
+}
+```
+
+## Draw 2D Texture
+Draw a 2D texture with graphics APIs of OpenTK, for more information you can reference to [OpenTK samples](https://github.sec.samsung.net/dotnet/opentk_samples).
+```C#
+private IGameWindow mainWindow;     // window
+
+private float[] vertices;           // vertex array
+private float[] textCoord;          // texture coordinate array
+
+private int mProgramHandle;         // programe handle
+
+protected override void OnCreate()
+{
+    mainWindow = Window;
+    mainWindow.RenderFrame += OnRenderFrame;
+}
+
+private void OnRenderFrame(Object sender, FrameEventArgs e)
+{
+    GL.Viewport(0, 0, mainWindow.Width, mainWindow.Height);
+
+    GL.ClearColor(Color4.CornflowerBlue);
+    GL.Enable(All.DepthTest);
+    GL.Clear(ClearBufferMask.ColorBufferBit | ClearBufferMask.DepthBufferBit);
+
+    GL.UseProgram(mProgramHandle);
+    positionLoc = GL.GetAttribLocation(mProgramHandle, "a_position");
+    texCoordLoc = GL.GetAttribLocation(mProgramHandle, "a_texCoord");
+
+    textureLoc = GL.GetUniformLocation(mProgramHandle, "s_texture");
+    GL.Uniform1(textureLoc, 0);
+
+    unsafe
+    {
+        fixed (float* pvertices = vertices)
+        {
+            // Prepare the triangle coordinate data
+            GL.VertexAttribPointer(positionLoc, 3, VertexAttribPointerType.Float, false, 6 * sizeof(float), new IntPtr(pvertices));
+            GL.EnableVertexAttribArray(positionLoc);
+        }
+        fixed (float* texCoord = textCoord)
+        {
+            GL.VertexAttribPointer(texCoordLoc, 2, VertexAttribPointerType.Float, false, 2 * sizeof(float), new IntPtr(texCoord));
+            GL.EnableVertexAttribArray(texCoordLoc);
+        }
+    }
+    mvpLoc = GL.GetUniformLocation(mProgramHandle, "u_mvpMatrix");
+
+    // Apply the projection and view transformation
+    GL.UniformMatrix4(mvpLoc, false, ref mvpMatrix);
+
+    GL.DrawArrays(All.Triangles, 0, 36);
+    GL.Finish();
+
+    // Disable vertex array
+    GL.DisableVertexAttribArray(positionLoc);
+    GL.DisableVertexAttribArray(inColorLoc);
+    mainWindow.SwapBuffers();
+}
+```
+
+## Sample Application
+
+We implemented an OpenTK sample application which using SkiaSharp to draw text on a rotating cube. For more information
+you can referecne to this [sample application](https://github.com/Samsung/Tizen-CSharp-Samples/tree/dev/TV/CubeWithSkiaSharp).
+
```

Comment:
Please change to:
An implementation of OpenTK sample app is made that uses SkiaSharp to draw text on a rotating cube.
For more information, see  [sample application](https://github.com/Samsung/Tizen-CSharp-Samples/tree/dev/TV/CubeWithSkiaSharp).